tests/app/oauth/test_rest.py

```diff
@@ -664,3 +664,13 @@ def test_returns_401_on_any_exception(self, client, mock_va_sso_authorize_access
         assert response.status_code == 401
         assert response.json == {"error": "Unauthorized", "description": "Authentication failure"}
         assert mock_statsd.incr.called_with('oauth.authorization.failure')
+
+
+class TestGetServicesByUser:
+    def test_should_return_200(self, client, mocker):
+        result = [{"id": '12345', "name": 'Test'}]
+        mocker.patch('app.dao.services_dao.dao_fetch_all_services_by_user', return_value=result)
+        response = client.get('/auth/my-services/12345')
+
+        assert response.status_code == 200
+        assert response.data == [{id: '12345', "name": 'Test'}]
```

Comment:
This test is failing because the GET request returns 404.  That happens because "12345" is not a valid user ID, and there is no matching endpoint.  Change it to something like "4bba25d4-fc93-4cc0-8d47-8e1d67151b67", and it should work.